mapserver/django/django_project/fba/models/all.py

```diff
@@ -354,7 +366,7 @@ class SubDistrict(base_model):
     prov_code = models.SmallIntegerField(blank=True, null=True)
     dc_code = models.SmallIntegerField(blank=True, null=True)
     name = models.CharField(max_length=255, blank=True, null=True)
-    sub_dc_code = models.DecimalField(primary_key=True, max_digits=65535, decimal_places=65535)
+    sub_dc_code = models.FloatField(primary_key=True)
```

Comment:
Had to change this to FloatField because it refuses to insert new rows due to decimal constraint.